### ISSUE TITLE
Built in PHP Server not respecting XDEBUG_CONFIG environment settings in yml

### ISSUE THREAD

**Issue by mccrodp** [RobLoach/drupal-fig#5]
Hi @RobLoach. It seems the following `XDEBUG_CONFIG` settings are ignored when running `docker-compose`

```
# Environment variables
  environment:
    # Drupal settings
    DRUPAL_PROFILE: standard
    DRUPAL_SITE_NAME: Drupal
    DRUPAL_USER: admin
    DRUPAL_PASS: admin
    DRUPAL_DBURL: mysql://drupal:drupal@mysql:3306/drupal
    XDEBUG_CONFIG: idekey=PHPSTORM
    XDEBUG_CONFIG: remote_enable=true
    XDEBUG_CONFIG: remote_connect_back=true
    XDEBUG_CONFIG: remote_autostart=true
```

What is the best way to get Xdebug working when using this env? Thanks!

**EDIT**: I guess if these are custom env vars, there is nothing in the Dockie image to receive these XDebug ones. If so, it'd be great if some xdebug vars like this could be added, or just advice on best practice for setting them in the dockie server/image. Many thanks.

**Comment by mccrodp** [comment]
I was able to get XDebug working eventually from host machine with PHPSTORM.

I added the following to the `web` container ini file _/etc/php5/cli/conf.d/dockie-dockie.ini_

```
xdebug.idekey = 'PHPSTORM'
xdebug.remote_enable = true
xdebug.remote_connect_back = true
```

It would be great if this were in place either by default in the `dockie` image and/or configurable via the environment variables listed above. In the meantime, the details in this comment may help people getting XDebug working.
